src/components/Assignment/index.ts

```diff
@@ -131,7 +131,6 @@ class Assignment extends BridgeBase {
                 .arNavigationSteps=${this.arNavigationSteps}
                 @MultiStepActionButtonClick="${this._onActionButtonClick}">
             </multi-step-component>
-            <lit-toast></lit-toast>
```

Comment:
We rely on <lit-toast> to display messages so, in the long term we can't just remove it. We should either get <lit-toast> to work or replace it with something. So, please leave a comment here to remind us to continue work on a suitable replacement. Thanks.

Reply:
Sure John, This Lit toast package was not upgraded since two years and the version we are using not supporting with Lit 2.0 version, need to find similar component to display toast messages which supports Lit 2.0, will create a story as well.